apps/aecore/src/aec_peer_connection.erl

```diff
@@ -711,23 +751,57 @@ handle_get_block_rsp(S, {get_block, From, _TRef}, Msg) ->
     end,
     remove_request_fld(S, get_block).
 
-%% -- Get Mempool --------------------------------------------------------------
+%% -- TX Pool ----------------------------------------------------------------
+
+handle_tx_pool_sync_init(S, _MsgObj) ->
+    case aec_tx_pool_sync:accept(peer_id(S), self()) of
+        ok ->
+            send_response(S, txps_init, {ok, #{}});
+        Err = {error, _} ->
+            send_response(S, txps_init, Err)
+    end,
+    S.
+
+handle_tx_pool_sync_unfold(S, MsgObj) ->
+    Unfolds = maps:get(unfolds, MsgObj, []),
```

Comment:
Similarly - why `[]`? Is this not guaranteed by https://github.com/aeternity/epoch/pull/1245/files#diff-c1692ca9a93deb9e4ca76237ebbc01fdR217 ?